releases/release-7.5.4.md

```diff
@@ -0,0 +1,125 @@
+---
+title: TiDB 7.5.4 Release Notes
+summary: Learn about the compatibility changes, improvements, and bug fixes in TiDB 7.5.4.
+---
+
+# TiDB 7.5.4 Release Notes
+
+Release date: xxxx, 2024
+
+TiDB version: 7.5.4
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.5/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v7.5/production-deployment-using-tiup)
+
+## Compatibility changes **tw@qiancai 1**
+
+- 在获取 ddl 历史任务时添加对获取任务数量的限制 [#55711](https://github.com/pingcap/tidb/issues/55711) @[joccau](https://github.com/joccau)
```

Comment:
```suggestion
- Limit the number of DDL historical tasks retrieved through the [TiDB HTTP API](https://github.com/pingcap/tidb/blob/release-7.5/docs/tidb_http_api.md) to 2048 by default to prevent OOM issues caused by an excessive number of historical tasks [#55711](https://github.com/pingcap/tidb/issues/55711) @[joccau](https://github.com/joccau)
```